var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -113,21 +114,24 @@ class Qgis(CMakePackage):
     depends_on("py-psycopg2", type=("build", "run"))  # TODO: is build dependency necessary?
     depends_on("py-pyqt4", when="@2")
     depends_on("py-pyqt5@5.3:", when="@3")
+    depends_on("py-sip", type="build", when="^py-sip@5:")
+    depends_on("py-pyqt-builder", type="build", when="^py-sip@5:")
```

Comment:
very strange, when I do `type="build", when="^py-sip@5:"` for py-sip and py-pyqt-builder cmake step fails. it's as if these packages aren't provided, hence no `sipbuild` command. Will revert these for now.